### PR TITLE
Added start parameters per route

### PR DESCRIPTION
**Pull request by sweiland-openrails**
See also https://github.com/openrails/openrails/pull/903/commits/a6a8a5396ec05b319379e4b694af29684ad9d13c

### CODE REVIEW

File: schema.json [hunk 1]
```diff
@@ -77,6 +77,39 @@
             "installSize": {
                 "type": "integer",
                 "minimum": 0
+            },
+            "start": {
+                "type": "object",
+                "required": [
+                    "locomotive"
```
Comment (cjakeman) [inline schema.json:84]: Why is only the "locomotive" a required field? On its own, that's not enough for a set of valid choices, I think.

**Review by cjakeman — COMMENTED** [review]
Hi Siebren,

It makes sense to have some default choices. I believe MSTS had an "Introductory Train Ride" as a starting point.

Can this be easily extended to a default Activity or a default Timetable too? (Not a priority just now.)

Best wishes,

Chris

**Comment by sweiland-openrails** [comment]
> Hi Siebren,
> 
> It makes sense to have some default choices. I believe MSTS had an "Introductory Train Ride" as a starting point.
> 
> Can this be easily extended to a default Activity or a default Timetable too? (Not a priority just now.)
> 
> Best wishes,
> 
> Chris

thanks for this reminder. I forgot to add them as required fields.

Siebren.

**Review by cjakeman — APPROVED** [review]
Hi Siebren,

This all looks good.

Approved with thanks,

Chris

**Pull request merged**